ci/win_test_installer.cmd

```diff
@@ -16,15 +16,23 @@ echo %NEURONHOME%
 :: If so, try again to generate it. No wait required like previous strategies, we rely on testing entropy from this point on.
 if not exist association.hoc.out (start /wait /REALTIME %cd%\ci\association.hoc)
 
+:: install oldest supported numpy
+C:\Python39\python.exe -m pip install numpy==1.20.3 || goto :error
+C:\Python310\python.exe -m pip install numpy==1.21.6 || goto :error
+C:\Python311\python.exe -m pip install numpy==1.23.5 || goto :error
+C:\Python312\python.exe -m pip install numpy==1.26.4 || goto :error
+
 :: test all pythons
 C:\Python38\python -c "import neuron; neuron.test(); quit()" || set "errorfound=y"
 C:\Python39\python -c "import neuron; neuron.test(); quit()" || set "errorfound=y"
 C:\Python310\python -c "import neuron; neuron.test(); quit()" || set "errorfound=y"
 C:\Python311\python -c "import neuron; neuron.test(); quit()" || set "errorfound=y"
-:: install numpy dependency
-python -m pip install "numpy<2"
+C:\Python312\python -c "import neuron; neuron.test(); quit()" || set "errorfound=y"
+C:\Python312\python -m pip list
 :: run also using whatever is system python
+python -m pip install numpy
```

Comment:
![image](https://github.com/user-attachments/assets/bc809253-17d0-4098-8426-6da63bd82dc3)

So far, I get this using the downloaded installer from the CI artifact... maybe we can produce a Debug artifact with more useful information.